### PR TITLE
cleanup launch agent cli a bit

### PR DESCRIPTION
**Pull request by stephchen**
the launch agent cli had both options for project/entity and an arg to specify the pair as a string. this changes it so project is now a required arg (which matches the ui experience a little more) and entity is an option that defaults to the current logged in user, ie `wandb launch-agent projectname` or `wandb launch-agent projectname --entity user`

### CODE REVIEW

File: wandb/sdk/internal/internal_api.py [hunk 1]
```diff
@@ -186,7 +186,7 @@ def app_url(self):
 
     @property
     def default_entity(self):
-        return self.settings("entity")
+        return self.viewer().get("entity")
```
Comment (KyleGoyette) [inline wandb/sdk/internal/internal_api.py:189]: Why is this change needed? Does it impact other parts of the CLI that use this function?

Comment (stephchen) [reply wandb/sdk/internal/internal_api.py:189]: the previous `self.settings("entity")` (which i think was attributed to you?) returned empty for me. as far as i can tell this fn isn't used anywhere but launch_agent

Comment (KyleGoyette) [reply wandb/sdk/internal/internal_api.py:189]: Oh, haha, I see. Okay, this looks good, matches the public api was well.

**Review by KyleGoyette — APPROVED** [review]
Nice!
